src/operator/nn/mkldnn/mkldnn_convolution.cc

```diff
@@ -430,8 +430,7 @@ void MKLDNNConvolutionForwardFullFeature(const MKLDNNConvFullParam &param, const
       weight.MKLDNNDataReorderAsync(fwd->GetPd().weights_desc());
       weight_mem = GetWeights(weight, fwd->GetPd().weights_desc(), param.conv_param.num_group);
     } else {
-      weight_mem = weight.GetMKLDNNData();
-      CHECK(weight_mem->get_desc() == fwd->GetPd().weights_desc());
+      weight_mem = weight.GetMKLDNNDataReorder(fwd->GetPd().weights_desc());
```

Comment:
Is this change required due to the new oneDNN version or is it a bugfix? I ask because we have a customer facing an issue with this check failing.

Reply:
Hi @josephevans , sorry for the late response, I must have overlooked your comment. This change is required due to the new oneDNN version, but may fix your customer issue as well. If you sent me a link to the issue I might be able to say something more.